slam_toolbox_ws/README.md

```diff
@@ -0,0 +1,31 @@
+# ROS2 SLAM-TOOLBOX
+
+### Run with docker
+
+```bash
+git clone https://github.com/j3soon/ros2-agv-essentials.git
+```
+
+```bash
+cd ros2-agv-essentials/slam_toolbox_ws/docker
+docker-compose pull
+docker-compose up -d --build
+```
+
+### Simulation with gazebo
+
+- Attach to the container
+  ```sh
+  docker attach ros2-slam-toolbox-ws
+  ```
+
```

Comment:
Since we didn't build this package in the Dockerfile, we need to run `colcon build` and `source $ROS2_WS/install/setup.bash` here. Otherwise, the next step will fail:

```
Package 'slam_simulation' not found: "package 'slam_simulation' not found, searching: ['/opt/ros/humble']"
```

P.S. Maybe we can consider running `colcon build` in `.bashrc`? I'm not sure about this though.

Reply:
I suggest implementing a straightforward conditional statement to verify whether the current folder corresponds to the workspace folder. If it does, we can execute 'colcon build' directly; otherwise, the script should do nothing preventing from building a wrong folder.

Or maybe we could directly add "colcon build" in README?

Reply:
Adding `colcon build` in README sounds simpler.